serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -79,7 +80,17 @@ minikube start [...] --driver podman
 
 === Install
 
-To install the {product_name} Operator, you can use the following command:
+To install the {product_name} Operator, you can use the following commands:
+
+.Install Cert Manager Operator
+[source,shell,subs="attributes+"]
+----
+kubectl apply -f {cert_manager_installer_url}
+----
+[NOTE]
+====
+The Cert Manager is recommended in order to help on abstracting the certificates configuration, if, for some reason it can't be installed, you need to make sure that the certificates needed by the Webhooks correctly configured, more information can be found link:{kubebuilder_webhooks_url}[here].
```

Comment:
or the user can deactivate the webhooks. I guess it should be documented to add a specific variable into the `operator.yaml` file or use kustomize. wdyt ?

Reply:
+1, will add this info after testing.